14/umbraco-cms/reference/management/services/README.md

```diff
@@ -1 +1,106 @@
+
+## [MemberService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IMemberService.html)
+
+## [MemberTypeService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IMemberTypeService.html)
+
+## [NotificationService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.INotificationService.html)
+
+## [PackagingService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IPackagingService.html)
+
+## [PublicAccessService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IPublicAccessService.html)
+
+## [RedirectUrlService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IRedirectUrlService.html)
+
+## [RelationService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IRelationService.html)
+
+## [ServerRegistrationService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IServerRegistrationService.html)
+
+## [TagService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.ITagService.html)
+
+## [TextService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.ILocalizedTextService.html)
+
+## [ContentService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IContentService.html)
+
+## [ContentTypeService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IContentTypeService.html)
+
+## [LocalizationService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.ILocalizationService.html)
+
+## [UserService](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.IUserService.html)
+
+You can find a list of all supported services in the [API Docs](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Core.Services.html).
+
+# Getting a Service
+
+All services can be accessed with the following using statement:
+
+```csharp
+using Umbraco.Cms.Core.Services;
+```
+
+In other cases, you may be able to use [Dependency Injection](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection). For instance if you have registered your own class in Umbraco's dependency injection, you can specify the service interface in your constructor.
+
+For example, if you use a `NotificationService` you can use Dependency Injection via the `INotificationService` interface like this:
+
+```csharp
+public class MyClass
+{
+    private INotificationService _notificationService;
+
+ public MyClass(INotificationService notificationService)
+ {
+  _notificationService = notificationService;
+ }
+}
+```
+
+In Razor views, you can access the Notification Service through the `@inject` directive:
+
+```csharp
+@inject INotificationService NotificationService
+```
+
+{% hint style="info" %}
+The same example can be used for the other service where you can replace the interface and service name.
+{% endhint %}
```

Comment:
```suggestion
You can use the above example for other services where you can replace the interface and service name.
```

I don't think this statement needs to be in a Note. There is nothing noteworthy (😝) to be highlighted to the readers.